### DISCUSSION TITLE
get CurrentUser Claim in custom entity

### DISCUSSION THREAD

**Discussion by tuohaibei** [abpframework/abp#17980] [Q&A] [answered]
Hi guys
Could i access currentuser claim in custom entity,
For example, is there a static class?
I have defined domain entity and it Inherited from BaseEntity,i want access current user  in baseEntity and get custom claim.
Is there a tool class here or how to operate it?

![image](https://github.com/abpframework/abp/assets/5082315/25531892-1409-4533-bc58-caaf8da944f0)

![image](https://github.com/abpframework/abp/assets/5082315/828b6776-8eaa-496c-9be7-21bc84fb68f0)

**Comment by maliming** [comment]
There is no way, You have to set it after creating the entity.

**Reply by tuohaibei** [reply, marked as answer]
thanks ,i have get user information at server layer,and passs it to entity.
